res/controllers/midi-components-0.0.js

```diff
@@ -318,21 +318,68 @@
         this.firstValueReceived = false;
     };
     Pot.prototype = new Component({
+        relative: false,
         inValueScale: function (value) { return value / this.max; },
-        input: function (channel, control, value, status, group) {
-            this.inSetParameter(this.inValueScale(value));
-            if (! this.firstValueReceived) {
-                this.firstValueReceived = true;
-                this.connect();
+        shift: function () {
+            if (this.relative) {
+                this.input = function (channel, control, value, status, group) {
```

Comment:
> Is this a bug or a feature?

This is a feature.

The use case I have in mind is using tempo faders with sync. Perhaps someone will come up with another use case, but nothing jumps to mind immediately.